src/components/modals/SupportRequestModal.js

```diff
@@ -310,48 +311,47 @@ export const SupportRequestModal = hh(
                 div({className: 'form-group first-form-group'}, [
                   label({id: 'lbl_attachment', className: 'common-color'},
                     ['Attachment']),
-                  <Dropzone
-                    onDrop={acceptedFiles => this.attachmentChangeHandler(
-                      acceptedFiles)}>
-                    {({isDragActive, openUploader, getRootProps, getInputProps}) => (
-                      section({
-                        style: {
-                          backgroundColor: this.state.attachment.length !== 0 ?
-                            'transparent' :
-                            (isDragActive ? '#6898c1' : '#ebecee'),
-                          fontSize: 14,
-                          lineHeight: '30px',
-                          paddingLeft: '1rem',
-                          display: 'flex',
-                          alignItems: 'center',
-                          border: this.state.attachment.length === 0 ?
-                            '1px dashed' :
-                            'none',
-                        },
-                      }, [
-                        <div {...getRootProps()}>
-                          <input {...getInputProps()} />
-                          <p>{this.state.attachment.length === 0 ?
+                  h(Dropzone, {
```

Comment:
Linter ended up being confused with the mix of jsx syntax and js syntax within this html return (didn't match expected format of array). While the error message wasn't entirely accurate, it still highlighted a syntax concern correctly.

Reply:
Scratch that, there's some styling issues cropping up from this conversion

Reply:
Found the issue, forgot to add the parenthesis at the end of the getInputProps call, styling is working as expected